masterlist.yaml

```
@@ -18799,32 +18889,48 @@ plugins:
       - crc: 0x0C6BB768
         util: 'SSEEdit v4.0.4'
 
+  - name: 'JK''s Temple of the Divines.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/57304/'
+        name: 'JK''s Temple of the Divines'
+    msg: [ *compatPatch_JKIPC ]
+
   - name: 'JK''s The Bannered Mare.esp'
     url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/33845/' ]
-    msg:
-      - <<: *patch3rdParty
-        subs:
-          - 'Immersive Citizens'
-          - '[JK''s Interiors Patch Collection](https://www.nexusmods.com/skyrimspecialedition/mods/35910/)'
-        condition: 'active("Immersive Citizens - AI Overhaul.esp") and not active("JKs Bannered Mare - Immersive Citizens Patch.esp")'
-      - <<: *patch3rdParty
-        subs:
-          - 'Open Cities Skyrim'
-          - '[JK''s Interiors Patch Collection OCS](https://www.nexusmods.com/skyrimspecialedition/mods/46114/)'
-        condition: 'active("Open Cities Skyrim.esp") and not active("JK''s Bannered Mare OCS Patch.esp")'
-      - <<: *patch3rdParty
-        subs:
-          - 'Skyrim Project Optimization'
-          - '[JK''s Interiors Patch Collection](https://www.nexusmods.com/skyrimspecialedition/mods/35910/)'
-        condition: 'active("Skyrim Project Optimization - .*\.esm") and not active("JKs Bannered Mare - SPO Patch.esp")'
+    msg: [ *compatPatch_JKIPC ]
```

JKIPC doesn't appear to contain a patch for OCS, so that message can't be replaced.

```suggestion
    msg:
      - *compatPatch_JKIPC
      - <<: *patch3rdParty
        subs:
          - 'Open Cities Skyrim'
          - '[JK''s Interiors Patch Collection OCS](https://www.nexusmods.com/skyrimspecialedition/mods/46114/)'
        condition: 'active("Open Cities Skyrim.esp") and not active("JK''s Bannered Mare OCS Patch.esp")'
```